src/utils/dateFormatter.ts

```diff
@@ -14,6 +14,16 @@ export const getLongDate = (date: string, locale: string | undefined | null): st
     timeZone: 'UTC',
   }).format(new Date(date));
 
+export const getLongDateTime = (date: string, locale: string | undefined | null): string =>
+  new Intl.DateTimeFormat(locale || 'en-US', {
+    month: 'long',
+    day: 'numeric',
+    year: 'numeric',
+    hour: 'numeric',
+    hour12: true,
+    timeZoneName: 'short',
+  }).format(new Date(date));
```

Comment:
The result from this function doesn't match the format in the designs 100%, but it's close. Still need to adjust the result from "March 16, 2024 at 3 PM HST" --> "March 16, 2024, 3pm HST".

Reply:
Here's an example of how we show similar dates in the app today (example Notifications dropdown) `March 14, 2024 at 4:38:49 PM`
Might be good to use a similar pattern, maybe suggest in the designs.